packages/xo-web/src/xo-app/backup/new/index.js

```diff
@@ -623,13 +625,13 @@ export default decorate([
       toggleDisplayAdvancedSettings: () => ({ displayAdvancedSettings }) => ({
         _displayAdvancedSettings: !displayAdvancedSettings,
       }),
-      setGlobalSettings: (_, { name, value }) => ({
+      setGlobalSettings: (_, globalSettings) => ({
         propSettings,
         settings = propSettings,
       }) => ({
         settings: settings.update('', setting => ({
           ...setting,
-          [name]: value,
+          ...globalSettings,
```

Comment:
I think, it's better to do this in an other PR.

Reply:
This is needed for this PR. I did it in a separated commits.

Reply:
You need this to pass more than 1 at a time, right?

I think @Rajaa-BARHTAOUI meant to do it in another PR *before this one*. "Make the change easy then make the easy change". But if we're out of time, I guess it's fine in this one.

Reply:
I thought we have two solutions. Make a change in an other PR if the current PR can be merged without it otherwise make it  in a different commit, no?

Reply:
Yes, it's fine.